hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/KeyGeneratorTestUtilities.java

```diff
@@ -65,4 +65,15 @@ public static Row getRow(GenericRecord record, Schema schema, StructType structT
     }
     return new GenericRowWithSchema(values, structType);
   }
+
+  public static Row genericRecordToRow(GenericRecord baseRecord, Schema schema, StructType structType) {
```

Comment:
can we add java docs for public methods. also, check if package private would suffice.